tensorflow/lite/tools/evaluation/proto/evaluation_stages.proto

```diff
@@ -121,6 +121,9 @@ message TfliteInferenceParams {
   // This helps benchmark cases where extensive pre-processing might not be
   // required for every input.
   optional int32 invocations_per_run = 4 [default = 1];
+
+  // nnapi_allow_fp16
+  optional bool nnapi_allow_fp16 = 5 [default = false];
```

Comment:
Drop this one? To support using a new field in this proto, I think more changes are needed. In this case, as the option is enabled by delegate_provider, we don't need this new field.

Reply:
yes, forget to delete this. Thanks